app/services/client/popup.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+class service {
+  getWindowSize() {
+    const dualScreenLeft = window.screenLeft !== undefined ? window.screenLeft : screen.left,
+      dualScreenTop = window.screenTop !== undefined ? window.screenTop : screen.top,
+      width = window.innerWidth || document.documentElement.clientWidth || screen.width,
+      height = window.innerHeight || document.documentElement.clientHeight || screen.height;
+
+    return { dualScreenLeft, dualScreenTop, height, width };
+  }
+
+  getCenterPosition(windowSize, dimensions) {
+    const left = windowSize.width / 2 - dimensions.w / 2 + windowSize.dualScreenLeft,
+      top = windowSize.height / 2 - dimensions.h / 2 + windowSize.dualScreenTop;
+
+    return { left, top };
+  }
+
+
+
+  /**
+ * openPopup Window
+ * @param {string} url - address of the popup page
+ * @param {object} dimensions { w: width of popup, h: height of popup}
+ */
```

Comment:
I know that this is being picky, but can you please align the asterisks in here? 😛 
```js
/**
 * openPopup Window
 * @param {string} url - address of the popup page
 * @param {object} dimensions { w: width of popup, h: height of popup}
 */
```